### ISSUE TITLE
Unmet peer dependency "moment" warning from `moment-timezone`

### ISSUE THREAD

**Issue by emattias** [fullcalendar/fullcalendar#6839]
### Reduced Test Case

"not reproducible with a link"

### Bug Description

We get

```
warning " > @fullcalendar/moment-timezone@5.11.0" has unmet peer dependency "moment@^2.29.1".
```
because we only have `moment-timezone` as dependency (which includes `moment`). Could [it](https://github.com/fullcalendar/fullcalendar/blob/master/packages/moment-timezone/package.json#L12) be removed?

### Screenshots
<img width="666" alt="CleanShot 2022-05-05 at 13 03 35@2x" src="https://user-images.githubusercontent.com/351537/166910685-daf533aa-9ff4-4062-9a37-936c3889a2e4.png">

**Comment by acerix** [comment]
I believe that's intentional to indicate which versions of `moment` is compatible with `@fullcalendar/moment-timezone`.

**Comment by arshaw** [comment]
Yes I agree @emattias. Since moment-timezone lists moment [as a normal dependency](https://cdn.jsdelivr.net/npm/moment-timezone@0.5.39/package.json), it is not the programmer's responsibility to install moment. Thus, it shouldn't be the programmer's responsibility to install moment for @fullcalendar/moment-timezone either

**Comment by arshaw** [comment]
Fixed in https://github.com/fullcalendar/fullcalendar/releases/tag/v6.0.0-beta.4

**Comment by arshaw** [comment]
Released in [v6.0.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.0.0)
